### PR TITLE
Extended the cargo sample with a simple HTTP api 

### PR DESCRIPTION
**Pull request by olger**
That allows fetching a specific piece of cargo.
Added a inmem view model to support the http fetch.
Added a sample test for the http api

BTW: the API will be extended with posting and all. Maybe a test with GraphQL ?

### CODE REVIEW

File: cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/httpapi/CargoRouteSpec.scala [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    override def send[T <: AggregateRootCommand](command: T)(implicit validator: ValidateableCommand[T]): Future[Unit] = ???
+
+    override def sendAndAsk[T <: AggregateRootCommand](command: T)(implicit validator: ValidateableCommand[T]): Future[_] = ???
+  }
+
+  val cargoRoute = new CargoRoute(commandGateway, cargoQueries)
+
+    "The Cargo route" should "fetch the data of a specific piece of cargo" in {
+      Get(s"/cargo/${cargoId1.id}") ~> Route.seal(cargoRoute.routes) ~> check {
+        status must be(StatusCodes.OK)
```
Comment (wketting) [inline cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/httpapi/CargoRouteSpec.scala]: If you test fetching specific data, you should validate the specific data is actually retrieved.

Comment (olger) [reply cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/httpapi/CargoRouteSpec.scala]: couldn't get the marshaling working.. so if you know how todo that :)

Comment (fmlrt) [reply cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/httpapi/CargoRouteSpec.scala]: there should be `SprayJsonSupport` in scope for that to work IIRC

**Comment by olger** [comment]
I've updated the code. Let me know your thoughts. 
There is a explicit TODO in de code that needs some separate thoughts outside of this pull request in order to keep things going.

**Pull request merged**